client/widgets/test-execution-time/time.template.html.slim

```diff
@@ -0,0 +1,30 @@
+.test-execution-time-widget.panel.panel-default
+  .panel-heading
+    help-button
+      p This widget shows the execution time for this specific test in chronological order (from older to newer). By default, only the latest {{ pageSize }} results are shown. If you choose to load older results, they will be added to the beginning of the curve as they are older. Only a maximum of {{ maxResults }} results can be shown.
```

Comment:
Replace first two sentences with "This widget shows the execution time history for this specific test in chronological order (from older runs to newer). By default, it only shows the execution time of the latest {{ pageSize }} results."